.goreleaser.yml

```diff
@@ -0,0 +1,15 @@
+# .goreleaser.yml
+# Build customization
+builds:
+  # makefile currently relies on this name
+  - binary: mobile-security-service-server
```

Comment:
It will be cool to name this somehow so it could look less like
![](https://i.pinimg.com/originals/1b/74/6b/1b746b5bc1c9b7a6b4579703a06bc3b9.png
)